src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -142,25 +262,84 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = memo(({ isLoading, onBack
   }, [isTradeQuoteRequestAborted, loadingSwappers, onBack])
 
   return (
-    <Box position='relative'>
+    <Flex position='relative' minHeight='full'>
       <Flex
         flexDir='column'
         width='full'
         px={2}
         pt={0}
-        pb={4}
         transitionProperty='max-height'
         transitionDuration='0.65s'
         transitionTimingFunction='ease-in-out'
         gap={2}
+        minHeight='full'
       >
-        <LayoutGroup>
-          <AnimatePresence>
-            {quotes}
-            {fetchingSwappers}
-          </AnimatePresence>
-        </LayoutGroup>
+        <Flex
+          // minHeight={`calc(100% - ${unavailableAccordionRef.current?.style.height}px)`}
```

Comment:
nitpick: 

```suggestion
```

Reply:
Damn, some dusts, removed in [2c0e7de](https://github.com/shapeshift/web/pull/8087/commits/2c0e7ded59e68034b730c8481c951c1f9dbf0bfa)